content/post/2017-04-28-daijiang-website.md

```diff
@@ -0,0 +1,95 @@
+
+Here, I briefly explain the structure of my website. Since I have very similar set up with [Yihui's website](https://yihui.name) and I used his [modified hugo-lithium-theme](https://github.com/yihui/hugo-lithium-theme)^[frankly, I cloned his website and then tweaked based on it, which I think is the easist way to get started. If you also want clone his or my website, I suggest to use `git clone --recursive url-of-github-repo`, which will also clone the theme for you.], I strongly recommend to read his [website set up](/2017/04/25/yihui-website/).
+
+Here is the tree structure of my website source code:
+
+```
+├── R
+│   ├── build.R
+│   ├── build_one.R
+│   └── clean_blogs.R
+├── README.md
+├── config.yaml
+├── content
+│   ├── 404.md
+│   ├── _index.md
+│   ├── about.md
+│   ├── cn
+│   ├── cn-about.md
+│   ├── cn-vitae.md
+│   ├── en
+│   ├── en-about.md
+│   ├── en-vitae.md
+│   ├── links.md
+│   ├── research.md
+│   ├── resume.md
+│   └── zoey
+├── index.Rmd
+├── layouts
+│   ├── _default
+│   └── partials
+├── static
+│   ├── CNAME
+│   ├── css
+│   ├── fonts
+│   ├── images
+│   ├── js
+│   └── pdf
+├── themes
+│   └── hugo-lithium-theme
+└── website.Rproj
+```
+
+Let's look at them individually. 
+
+- `R` folder: this folder hosts R scripts. The `build.R` and `build_one.R` were cloned from Yihui and they are used to convert `*.Rmd` to `*.md`. I did not use them yet. The `clean_blogs.R` was written by me and mainly to change yaml headers from `jekyll`. So, basically, you can ignore this folder.
+- `README.md`: some explanation for github repository.
+- `config.yaml`: this is the main file where you set up things. It is pretty straightforward but you need be careful to put all your information correct, e.g. Disqus shortname, Twitter handle, etc.
+- `content` folder: this the folder to put your webpages and blog posts. For example, I have `_index.md`, `about.md`, `resume.md`, `research.md`, and three subfolders that will be turned into three blogs. If you open my [website](https://daijiang.name), it will be clear what roles these files/folders have. You can order them in the `config.yaml` file with the `weight` option^[start with 1 instead of 0.].
+- `index.Rmd`: you do not need to change it.
+- `layouts` folder: this folder has files that will control the details of the website.
```

Comment:
I'm not sure why but the bullet for `layouts` is appearing in a line above instead of the right place
![image](https://cloud.githubusercontent.com/assets/1043111/25552941/b3b7b69a-2c74-11e7-8fd9-9228d8be8e75.png)